go.mod

```diff
@@ -83,6 +85,7 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/rivo/uniseg v0.4.7 // indirect
+	github.com/vbauerster/mpb/v8 v8.8.3
```

Comment:
Can you check why this direct module is mentioned in this indirect section? it should be in the section where all the other direct dependencies are defined!